Classes/Command/PixxioCommandController.php

```diff
@@ -195,4 +224,80 @@ public function updateMetadataCommand(string $assetSource = 'flownative-pixxio',
             !$quiet && $this->outputLine('   in order to make changes visible in the frontend');
         }
     }
+
+    /**
+     * Import pixx.io categories as asset collections
+     *
+     * @param string $assetSourceIdentifier Name of the pixx.io asset source (defaults to "flownative-pixxio")
+     * @param bool $quiet If set, only errors will be displayed.
+     * @param bool $dryRun If set, no changes will be made.
+     * @return void
+     * @throws IllegalObjectTypeException
+     * @throws \Flownative\Pixxio\Exception\ConnectionException
+     */
+    public function importCategoriesAsCollectionsCommand(string $assetSourceIdentifier = 'flownative-pixxio', bool $quiet = true, bool $dryRun = false): void
```

Comment:
I think $quiet should be false by default.

Reply:
I'll keep it true by default for two reasons:

- consistency with the Canto asset source CLI behaviour
- this is supposed to be run in a cron job, so quiet is the more probably case (IMHO)